test/common/components/SignInLink.unit.spec.jsx

```diff
@@ -100,7 +100,7 @@ describe('<SignInLink>', () => {
   // TODO: While this _does_ test that the function is called, it doesn't test
   //  that the function is called _at the right time_. We should circle back to
   //  this and fix that.
-  it('should call onLogin after a successful login', () => {
+  xit('should call onLogin after a successful login', () => {
```

Comment:
This test is async but isn't written properly. I'm excluding it, since it causes confusion when some other tests fail.